### DISCUSSION TITLE
Integrare allarme Silentron

### DISCUSSION THREAD

**Discussion by NiKo-OnGit** [indomus/forum#2955] [GENERALE - Discussioni generiche]
Ciao a tutti,
vorrei integrare in HA una centrale di allarme Silentron modello Silenya 5500 HT.
Attualmente la centrale funziona solo via radio con i vari sensori, ma ha la morsettiera, quindi ipotizzavo si potesse collegare un relé o altro per controllarla.
Non ho bisogno di funzionalità particolari, mi serve solo l'attivazione/disattivazione totale ed avere modo di vederne lo stato (attivo/disattivo), poi se eventualmente è possibile avere qualcosa in più tipo l'avviso nel momento in cui l'allarme scatta o l'attivazione delle singole zone, ben venga, ma non è indispensabile.
Avete qualche suggerimento?

**Comment by lux90ita** [comment]
ingressi
via filo: 6 linee a doppio bilanciamento (1-6), programmabili liberamente sulle 6 zone A B C P R T - 1 ingresso bilanciato per segnalazione GUASTO (24h) 
Uscite
due relè programmabili per funzioni varie a scambio libero max 500mA 12V.

bisogna vedere cosa ti fà programmare la centrale
però avendo solo 2 uscitè sei un pò limitato

ma non riesci ad accedere via web o app?
